packages/web-components/src/components/va-privacy-agreement/va-privacy-agreement.tsx

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable i18next/no-literal-string */
+import { Component, Host, h, Prop, Event, EventEmitter, Fragment } from '@stencil/core';
+
+/**
+ * @componentName Privacy agreement
+ * @maturityCategory caution
+ * @maturityLevel candidate
+ */
+@Component({
+  tag: 'va-privacy-agreement',
+  styleUrl: 'va-privacy-agreement.css',
+  shadow: true,
+})
+export class VaPrivacyAgreement {
+  /**
+   * Whether the checkbox is checked or not.
+   */
+  @Prop({ mutable: true }) checked?: boolean = false;
+
+  /**
+   * Whether to display the error message or not.
+   */
+  @Prop() isError?: boolean = false;
```

Comment:
Do we need an external prop for this? I'm wondering if a State variable could work.

Reply:
I considered that but there were two things that made me pause; we would probably still need to communicate back what the error state is for any kind of form or page validation handling and second thing is that it would cause a rerender so I wasn't sure if that was worth doing. 

I do like the idea of delegating error management to the component but wouldn't the developer implementing it still need to know the error state either way?

Reply:
You're right about the `@State` triggering a rerender - we could just have it be a class property instead if we wanted to avoid that. 

We've already got the `checked` prop, so could that be used to communicate an error state?